13_3_Setting_Up_a_Channel.md

```diff
@@ -0,0 +1,260 @@
+# 13.3: Setting Up a Channel
+
+> :information_source: **NOTE:** This is a draft in progress, so that I can get some feedback from early reviewers. It is not yet ready for learning.
+
+You're now ready to your first lightning network channel. To begin with, you'll need to understand what is and how it's created an c-lightning channel.
+
+## What is a channel
+
+In simple terms a lightning channel is a money tube that allows fast, cheap and private transfers of money without sending transactions to the blockchain. 
+More technically a channel is a 2-of-2 multisignature bitcoin onchain transaction that establishes a financial relationship between two agents without trust.
+As we seen in [§6.1:Sending a transaction to a multisig](06_1_Sending_a_Transaction_to_a_Multisig.md) chapter multisignatures are generally described as being "m of n". That means that the transaction is locked with a group of "n" keys, but only "m" of them are required to unlock the transaction.   
+
+Channels on the Lightning Network always are created between two nodes. The channel mantains a local database with bitcoin balance for both parts keeping track of how much money they each have.
+
+In this chapter we will use testnet network and will use c-lightning as primary node to show all processes related but we will use a lnd node too to create a channel with it.  If you're running our BitcoinStandup setup you could choose either of the two implementations at the time of installing.
+
+### Steps to create a channel
+
+* Fund your c-lightning wallet with some satoshis.
+* Connect to remote node as a peer.
+* Open channel.
+
+#### Fund you c-lightning wallet.
+
+The first thing you need to do is send some satoshis to your c-lightning wallet.  You can create a new address using  `lightning-cli newaddr` command to use it later. The newaddr RPC command generates a new address which can subsequently be used to fund channels managed by the c-lightning node.  This last transaction is called the [funding transaction](https://github.com/lightningnetwork/lightning-rfc/blob/master/03-transactions.md#funding-transaction-output) and it needs to be confirmed before funds can be used.  You can specify the type of address wanted,  if not specified the address generated will be a bench32.
```

Comment:
> :book: ***What is a c-lightning wallet?***

This is a great place to explain that it's a wallet that c-lightning will use to fund channels, not an actual channel.

Reply:
Done.